debian/rules

```diff
@@ -13,8 +13,8 @@ override_dh_auto_configure:
 		--with-twitter-consumer-secret="rbUEJCBEokMnGZd8bubd0QL2cSmoCjJeyiSJpnx3OM0" \
 		--with-facebook-client-id="271112146382618" \
 		--with-foursquare-client-id="1I2UNJXPHNDZT3OPZOOA5LCPIUEUJFMKRXSF42UFCN1KXKTK" \
-		--with-google-client-id="1032863574955-rmcjs0d9ulj5h5mhj0hrt1t3jf74at00.apps.googleusercontent.com" \
-		--with-google-client-secret="qFyrW7sAct8q2PCp5mHBPtPg" \
+		--with-google-client-id="1032863574955-44edra7uap3789kuc3cmplofd620ombh.apps.googleusercontent.com" \
+		--with-google-client-secret="QvEYYpxvrn6Oo-mOWZ_obhre" \
```

Comment:
Where do these come from?

Reply:
I had to register a new pair of keys, for the "desktop application" client type.